massa-versioning/src/versioning.rs

```diff
@@ -1296,6 +1309,61 @@ impl MipStoreRaw {
         Ok(())
     }
 
+    // Final state
+    pub fn is_key_value_valid(&self, serialized_key: &[u8], serialized_value: &[u8]) -> bool {
+        self._is_key_value_valid(serialized_key, serialized_value)
+            .is_ok()
+    }
+
+    pub fn _is_key_value_valid(
```

Comment:
I'd vote for a different name `verify_key_value_validity` or maybe `check_key_value`
(I don't like much to have that two functions that differ only by a `_` in their name)

Reply:
this is the very same function and _is_key_value_valid only exists for and if in unit tests we'd want to check for a particular error...

Reply:
So shouldn't it be scoped for test non `pub`?
With `#[cfg(feature = "test-exports")]` the intent would be clearer.